### PR TITLE
Add information about RCS channel support from conversation API persp…

### PR DESCRIPTION
**Pull request by Filip-Kus**
Add information about RCS channel support from conversation API perspective.

### CODE REVIEW

File: docs/conversation/conversation-channel-support.md [hunk 1]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+To start using RCS through Conversation API you need to first have a Sinch RCS bot -
+[read more](https://developers.sinch.com/docs/rcs-introduction).
+
+#### Channel Configuration
+
+##### Sending Config
+
+Sending a RCS message requires a Conversation API **app** with
+`channel_credentials` for RCS channel (You can create an **app** either in the 
+portal or thorough API call - [read more](https://developers.sinch.com/reference/#app_createapp)). Example, **app** is given in the following snippet:
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2237]: *through

File: docs/conversation/conversation-channel-support.md [hunk 2]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+- `{{RCS_BEARER_TOKEN}}` with the bot's access token.
+
+##### Receiving Config
+
+Receiving contact messages (replies) and delivery receipts
+from RCS requires setting the Callback URL of the RCS bot to
+point to your Conversation API **app**. The URL is the following:
+
+```
+https://rcs-adapter.conversation-api.int.prod.sinch.com/adapter/v1/{{CONV_API_APP_ID}}/callback
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2265]: region support! please add that they should either use eu1 or us1 corresponding to the App´s region

File: docs/conversation/conversation-channel-support.md [hunk 3]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+```
+
+Where:
+
+- `{{CONV_API_APP_ID}}` is your **app** id.
+
+You also need to configure at least one Conversation API webhook which
+will trigger POST callbacks to the given URL (You can create a **webhook** 
+either in the portal or thorough API call - [read more](https://developers.sinch.com/reference/#webhooks_createwebhook)).
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2274]: *through

File: docs/conversation/conversation-channel-support.md [hunk 4]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+
+##### Choice Messages
+
+RCS channel provides native support for Choice Messages. As a user you can select from four choice types:
+- Text Choice
+- URL Choice
+- Call Choice
+- Location Choice  
+
+Below snippets presents all representations of above choice types. You can send at least 3 choices in one message.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2361]: hehe, not at least, but maximum :D

File: docs/conversation/conversation-channel-support.md [hunk 5]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+    }
+  }
+}
+```
+
+The rendered message:
+
+![Media Message](images/channel-support/rcs/RCS_Video_Media_Message.jpg)
+
+##### Choice Messages
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2353]: one more #

File: docs/conversation/conversation-channel-support.md [hunk 6]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+}
+```
+
+The rendered message:
+
+![Choice Message](images/channel-support/rcs/RCS_Call_Location_Choice_Message.jpg)
+
+###### Card Messages
+
+RCS supports natively Card Messages with at least 3 choices.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2436]: same here, max 3 not at least :D

File: docs/conversation/conversation-channel-support.md [hunk 7]
```diff
@@ -2221,3 +2221,583 @@ Conversation API POST to `MESSAGE_DELIVERY` webhook:
...
+}
+```
+
+The rendered message:
+
+![Card Message With URL Choice](images/channel-support/rcs/RCS_Card_Message.jpg)
+
+###### Carousel Messages
+
+RCS supports natively Carousel Messages. You can put from 1 to 10 cards in one message.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:2487]: please mention the outer choices too!

File: docs/conversation/conversation-channel-support.md (hunk 1 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: changed

File: docs/conversation/conversation-channel-support.md (hunk 2 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: Nice find, added.

File: docs/conversation/conversation-channel-support.md (hunk 3 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: changed.

File: docs/conversation/conversation-channel-support.md (hunk 4 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: 🤦 changed.

File: docs/conversation/conversation-channel-support.md (hunk 5 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: Yepp, changed.

File: docs/conversation/conversation-channel-support.md (hunk 6 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md]: CHanged.

File: docs/conversation/conversation-channel-support.md (hunk 7 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support.md:2488]: I added sth like that: "Additionally, RCS channel supports maximum 3 outer choices. You can put outer choices into 
the `carousel_message.choices` array field, and these choices will be rendered right after the displayed cards."

**Pull request merged**